engine/src/main/java/org/terasology/persistence/serializers/ProtobufSerializer.java

```diff
@@ -30,42 +30,101 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
-// TODO: Document
```

Comment:
All the javadoc for the methods looks good. I'd just like to see some on the class itself stating what it does. If you need help please ask on the Discord and a mentor can give you some assistance